tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/uniontypes/UnionTypeTest.java

```diff
@@ -125,4 +125,13 @@ public void testAmbiguousAssignment() {
         BAssertUtil.validateError(negativeResult, i++, "ambiguous type '(ClosedBar|OpenBar)'", 44, 28);
         BAssertUtil.validateError(negativeResult, i++, "incompatible types: expected 'string', found 'int'", 47, 31);
     }
+
+    @Test(description = "Test nullable check")
+    public void testNullableCheck() {
+        try {
+            BRunUtil.invoke(result, "testNullableTypeArrayFill");
+        } catch (Throwable e) {
+            Assert.fail("Nullable check in Union type failed");
```

Comment:
Do we throw any exception here?

Reply:
Instead of using a try-catch, shall we break the test in to two tests (a positive and a negative one which panics). Then can check for the expected runtime exception in the negative one.